packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,260 @@
+                        <Icon
+                          iconName="DtmfDialpadButton"
+                          style={{ color: theme.palette.themePrimary, padding: '8px' }}
+                        />
+                      </Stack>
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalMeetingId}</Text>
+                    </Stack.Item>
+                  </Stack>
+                </Stack.Item>
+                <Text className={phoneInfoTextStyle}>{formatMeetingId(conferencePhoneInfoList[0].conferenceId)}</Text>
+              </Stack>
+              <Stack horizontal horizontalAlign="space-between" verticalAlign="center">
+                <Stack horizontal>
+                  <Stack.Item className={phoneInfoIcon(theme)}>
+                    <Icon iconName="PhoneInfoWait" style={{ color: theme.palette.themePrimary, padding: '8px' }} />
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalWait}</Text>
+                  </Stack.Item>
+                </Stack>
+              </Stack>
+            </Stack>
+          )}
+        </Modal>
+      }
+    </>
+  );
+};
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumber = (phoneNumber: string): string => {
+  if (!phoneNumber) {
+    return '';
+  }
+  let enchantedPhoneNumber = phoneNumber;
+  if (!phoneNumber.startsWith('+')) {
+    enchantedPhoneNumber = `+${phoneNumber}`;
+  }
+  if (isPossiblePhoneNumber(enchantedPhoneNumber)) {
+    return parsePhoneNumber(enchantedPhoneNumber)?.formatInternational() || enchantedPhoneNumber;
+  }
+  return phoneNumber;
+};
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumberInfo = (
+  phoneNumber: ConferencePhoneInfo | undefined,
+  strings: MeetingConferencePhoneInfoModalStrings | undefined
+): string => {
+  if (!phoneNumber) {
+    return '';
+  }
+  const templateText = phoneNumber.isTollFree
+    ? strings?.meetingConferencePhoneInfoModalTollFree
+    : strings?.meetingConferencePhoneInfoModalToll;
+  return (
+    templateText
+      ?.replace('{phoneNumber}', formatPhoneNumber(phoneNumber.phoneNumber))
+      .replace('{country}', phoneNumber.country || '')
+      .replace('{city}', phoneNumber.city || '')
+      .trim() || ''
+  );
+};
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * format meeting id
+ */
+export const formatMeetingId = (meetingId?: string): string => {
+  if (!meetingId) {
+    return '';
+  }
+  if (meetingId?.length !== 9) {
+    return meetingId;
+  }
+
+  return [meetingId.slice(0, 3), meetingId.slice(3, 6), meetingId.slice(6, 9)].join(' ') + '#';
```

Comment:
Out of interest, why do we add the `#`?

Reply:
Oh is this the phone equivalent of pressing enter?

Reply:
Yes, you need to hit # at the end of conference code to continue